packages/testsuite/cypress/e2e/metrics/test-configuration-subsystem-metrics.cy.ts

```diff
@@ -0,0 +1,71 @@
+describe("TESTS: Configuration => Subsystem => Metrics", () => {
+  const address = ["subsystem", "metrics"];
+  const configurationFormId = "model-browser-model-browser-root-form";
+
+  const exposedSubsystems = "exposed-subsystems";
+  const exposedSubsystemsDefaultValue = "*";
+  const exposedSubsystemsCustomValue = "subsystems-to-update";
+  const prefix = "prefix";
+  const customPrefixValue = "wildfly.metrics.prefix:custom_prefix";
+  const securityEnabled = "security-enabled";
+
+  let managementEndpoint: string;
+
+  before(() => {
+    cy.startWildflyContainer().then((result) => {
+      managementEndpoint = result as string;
+    });
+  });
+
+  after(() => {
+    cy.task("stop:containers");
+  });
+
+  it("Edit exposed subsystems", () => {
+    cy.navigateToGenericSubsystemPage(managementEndpoint, address);
+    cy.editForm(configurationFormId);
+    cy.clearSelectedItems('div.tag-manager-container')
+    cy.formInput(configurationFormId, exposedSubsystems)
+      .clear()
+      .type(exposedSubsystemsCustomValue + "{enter}")
+      .trigger("change");
+    cy.saveForm(configurationFormId);
+    cy.verifySuccess();
+    cy.verifyListAttributeContains(managementEndpoint, address, exposedSubsystems, exposedSubsystemsCustomValue);
+    cy.verifyListAttributeDoesNotContain(managementEndpoint, address, exposedSubsystems, exposedSubsystemsDefaultValue);
+  })
+
+  it("Edit prefix", () => {
+    cy.navigateToGenericSubsystemPage(managementEndpoint, address);
+    cy.editForm(configurationFormId);
+    cy.text(configurationFormId, prefix, customPrefixValue, true);
+    cy.saveForm(configurationFormId);
+    cy.verifySuccess();
+    cy.verifyAttribute(managementEndpoint, address, prefix, customPrefixValue);
+  })
+
+  it("Toggle security-enabled", () => {
+    let value: boolean;
+    cy.task("execute:cli", {
```

Comment:
@zhantemirov Hi, here you can use method `getDefaultBooleanValue`

https://github.com/hal/berg/blob/main/packages/testsuite/cypress/support/resource-utils.ts#L150

here is also usage
https://github.com/hal/berg/blob/main/packages/testsuite/cypress/e2e/io/test-configuration-subsystem-io-buffer-pool.cy.ts#L82

Reply:
Should be OK now, thanks